### PR TITLE
Add target organism lineage to shipping.presenc...

### PR DESCRIPTION
**Pull request by kairstenfay**
e_absence_result_v1

### CODE REVIEW

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -108,11 +108,13 @@ create or replace view shipping.presence_absence_result_v1 as
...
     select sample.identifier as sample,
            target.identifier as target,
-           present
+           present,
+           organism.lineage
 
     from warehouse.sample
     join warehouse.presence_absence using (sample_id)
     join warehouse.target using (target_id)
+    join warehouse.organism using (organism_id)
```
Comment (tsibley) [inline schema/deploy/shipping/views.sql]: This needs to be a left join so we don't exclude results for targets that aren't yet linked to an organism.

Comment (kairstenfay) [reply schema/deploy/shipping/views.sql]: Done.

**Comment by kairstenfay** [comment]
[Add target organism lineage to shipping.presence_absence_result_v1](https://trello.com/c/0qH0FR5n/215-add-target-organism-lineage-to-shippingpresenceabsenceresultv1)

**Comment by tsibley** [comment]
I renamed the view column from `lineage` → `organism` to match our conventions so far (e.g. `target.identifier` is named just `target` in most views).

Merged and deployed.

**Pull request merged**